app/workers/rent_due_notifier.rb

```diff
@@ -0,0 +1,15 @@
+class RentDueNotifier
+  include Sidekiq::Worker
+
+  def perform
+    Rent.where(id: rent_ids).find_each do |rent|
```

Comment:
Why are you doing a where here? You already do it in `rent_ids` method

Reply:
@facucompe My bad! At some point during development I was returning rent_ids and using them in perform, then I realized it was not necessary and removed the `.ids` part in the rent_ids method, but forgot to change the rest.